### ISSUE TITLE
Index sidebar breaks showing only parts of the notes, and repeating some

### ISSUE THREAD

**Issue by quesada** [pfalcon/zim-desktop-wiki#13]
![image](https://user-images.githubusercontent.com/182207/152607935-0883552d-43c6-4d29-b530-ba4d4d1c6588.png)
This is an example. 

Not sure how I triggered it. I tried to Jump using ctrl J to a page. This page didn't show in the index (weird) so I tried to scroll manually to find it alphabetically. Then the index sidebar got glitchy.

This is on .73.5, which is the current one in this repo as of 04.02.2022

**Comment by pfalcon** [comment]
That's likely https://github.com/pfalcon/zim-desktop-wiki-markdown/issues/4.

**Comment by quesada** [comment]
You are right, sorry I didn't read it before :(. Great that there's a workaround!
